scenarios/kube/composite.schema.json

```diff
@@ -0,0 +1,89 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "$id": "https://github.com/chaos-kubox/krkn/",
+    "type": "object",
+    "default": {},
+    "title": "Composite scenario for Krkn",
+    "required": [
+        "steps"
+    ],
+    "properties": {
+        "iterations": {
+            "type": "integer",
+            "default": 1,
+            "title": "How many iterations to execute",
+            "examples": [
+                3
+            ]
+        },
+        "steps": {
+            "type": "array",
+            "default": [],
+            "title": "The steps Schema",
+            "items": {
+                "type": "object",
+                "default": {},
+                "title": "A Schema",
+                "required": [
+                    "pod"
+                ],
+                "properties": {
+                    "pod": {
+                        "type": "object",
+                        "default": {},
+                        "title": "The pod Schema",
+                        "required": [
+                            "name_pattern",
+                            "namespace_pattern"
+                        ],
+                        "properties": {
+                            "name_pattern": {
+                                "type": "string",
+                                "default": "",
+                                "title": "The name_pattern Schema",
+                                "examples": [
+                                    ""
+                                ]
+                            },
+                            "namespace_pattern": {
+                                "type": "string",
+                                "default": "",
+                                "title": "The namespace_pattern Schema",
+                                "examples": [
+                                    ""
+                                ]
+                            }
+                        },
+                        "examples": [{
+                        "name_pattern": "test-.*",
+                        "namespace_pattern": "default"
+                        }]
+                    }
+                },
+                "examples": [{
+                    "pod": {
+                        "name_pattern": "test-.*",
+                        "namespace_pattern": "default"
+                    }
+                }]
+            },
+            "examples": [
+                [{
+                    "pod": {
+                        "name_pattern": "test-.*",
+                        "namespace_pattern": "default"
+                    }
+                }]
+            ]
+        }
+    },
+    "examples": [{
+        "iterations": 1,
+        "steps": [{
+            "pod": {
+                "name_pattern": "test-.*",
+                "namespace_pattern": "default"
+            }
+        }]
+    }]
+}
```

Comment:
Please add newline at end of file

Reply:
Why is that necessary in 2022?

Reply:
new line character is line termination. Without it the last line of the file is not terminated.
It can cause issues with some tooling including git blame.
There are several discussions about it , for reference: 
* https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b
* https://en.wikipedia.org/wiki/Newline

Reply:
I just checked, none of Vim, Emacs, IntelliJ+friends, Atom, VS Code, TextMate, and Sublime require an exlicit setting to be enabled for automatically adding a newline, which is just as bad as reformatting a file in an unrelated PR. This newline thing is, as far as I can tell, outdated and no longer true.